### ISSUE TITLE
Add Jupyter Comms as a supported backend

### ISSUE THREAD

**Issue by banesullivan** [Kitware/wslink#129]
`aiohttp` is the only currently supported backend for `wslink`. Could we use the Jupyter comms implementation as a "backend" for at least the client-server communication?

This would prevent the need for using `jupyter-server-proxy` with `wslink` and `trame` to view Trame-based UIs in Jupyter.

**Comment by banesullivan** [comment]
Maybe AnyWidget could be a more viable and robust route: https://anywidget.dev/blog/introducing-anywidget/

**Comment by jourdain** [comment]
This is even further apart from what wslink offers...

**Comment by banesullivan** [comment]
Well shucks...

**Comment by jourdain** [comment]
You think UI instead of network.

**Comment by banesullivan** [comment]
Yes -- still wrapping my head around anywidget (just stumbled upon this in the last 10 minutes). But I think there may be a way to leverage it to work around the Jupyter issues we face with jupyter-server-proxy. I'll investigate and try to report back when I have the time
